src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs

```diff
@@ -524,7 +524,7 @@ private static bool StatementClosingDelimiterIsMissing(SyntaxNode currentNode)
         /// </returns>
         private static bool RequiredDelimiterIsMissing(SyntaxNode currentNode)
         {
-            return currentNode.GetBrackets().closeBrace.IsMissing ||
+            return currentNode.GetBrackets().closeBracket.IsMissing ||
```

Comment:
port of https://github.com/dotnet/roslyn/pull/52704 into this branch to fix build beak.